packages/Filter/src/filter.scss

```diff
@@ -0,0 +1,106 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-filter-background: #{tokens.$grey-200};
+  --ids-filter-border-radius: calc(#{tokens.$border-radius-md} * 4);
+  --ids-filter-color: #{tokens.$grey-600};
+  --ids-filter-font-family: #{tokens.$primary-font-family};
+  --ids-filter-font-size: #{tokens.$font-size-3};
+  --ids-filter-line-height: #{tokens.$line-height-xxl};
+  --ids-filter-padding: #{tokens.$space-2} #{tokens.$space-3};
+
+  /* Hover */
+  --ids-filter-background-hover: #{tokens.$grey-300};
+
+  /* Selected */
+  --ids-filter-background-selected: #{tokens.$grey-500};
+  --ids-filter-color-selected: #{tokens.$samoyed};
+
+  /* Disabled */
+  --ids-filter-color-disabled: #{tokens.$grey-400};
+
+  /* Focus */
+
+  /* Needed to add a 0.1px blur to fix a bug where
+  chrome would have a lingering blue line when focusing in and out */
+  --ids-filter-focus: 0 0 0 2px #fff, 0 0 0.1px 3.5px #0c46d0;
```

Comment:
We had fix that same issue in the `Button` maybe you can use the same solution here.

By memory it was something with also using:
```css
outline: 0;
```
in the CSS properties.

Reply:
sadly this isn't the same thing. It's specifically due to the box-shadow. It's like some sort of glitch on chrome.
This is what it looks like if you're tabbing on the filter:
![image](https://user-images.githubusercontent.com/110498164/192856962-9b9af831-7c8b-43e2-add2-19b5452f70ad.png)